blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -796,9 +808,8 @@ public void testCopyBlobReplaceMetadata() throws Exception {
          } finally {
             Closeables2.closeQuietly(is);
          }
-         // TODO: S3 overrideMetadataWith also overrides system metadata
-         // TODO: Swift does not preserve system metadata
-         //checkContentMetadata(toBlob);
+         // TODO: Swift copy object *appends* user metadata, does not overwrite
```

Comment:
I am not completely sure. Shouldn't we be trying to have the same outcome? The question is if metadata should be completely overridden or appended when using the abstraction, right?